.stylelintrc.js

```diff
@@ -42,7 +43,7 @@ module.exports = {
             }
         ],
         'max-nesting-depth': [
-            3,
+            4,
```

Comment:
Bumped slightly to resolve a lot of issues and avoid having to add a ignore comments for all of them.
Respecting this rule retrospectively would require refactoring the scss in a way that could result in unexpected changes